.github/workflows/publish-release.yml

```diff
@@ -1,46 +1,50 @@
-name: Generate new release
+# Workflow base: https://semantic-release.gitbook.io/semantic-release/recipes/ci-configurations/github-actions
+
+name: Publish release
+
 on:
-  push:
-    tags:
-      - 'v1*'
+  workflow_run:
+    workflows: ["Run tests"]
+    branches:
```

Comment:
Bisogna verificare che questo workflow venga attivato al completamento del workflow "Run tests" ma solo al push sulla main e NON nelle PR

Reply:
@astagi stavo considerando che probabilmente conviene attivare questo workflow manualmente, per due motivi:

1. Se ad esempio ho più merge da effettuare e voglio fare solo un rilascio, oppure non voglio rilasciare subito una modifica che subirà altri cambiamenti
2. Per sicurezza. Ad esempio si fa un push sulla *main* in modo involontario.

Dimmi tu cosa preferisci.
Per rendere manuale questo workflow (e quindi le catene di workflow legate a questo) basta cambiare la regola in `workflow_dispatch` e quindi il workflow dovrà essere avviato manualmente dalla sezione "Action"

Reply:
@astagi ho impostato `workflow_dispatch` così da eseguirlo manualmente (solo da branch main)